README.rst

```diff
@@ -48,6 +48,82 @@ We use submodules to include external libraries in sympy-live::
 This is sufficient to clone appropriate repositories in correct versions
 into sympy-live (see git documentation on submodules for information).
 
+Setting Up for Testing PR 144
+-----------------------------
+Here are the steps to be followed for testing the PR 
+https://github.com/sympy/sympy-live/pull/144. The following are 
+the steps to be followed for Ubuntu. The steps for other Operating Systems 
+could be reproduced similiarly.
+
+You must have python3 installed. You can check your version by::
+
+    $ python3 -V
+
+The version must be greater than 3.4. If not present you can install it by::
+
+    $ sudo apt install python3
+
+You need to create the virtual environment and activate it::
+
+    $ sudo apt-get install python3-venv
+    $ python3 -m venv env
+    $ source env/bin/activate
+
+SymPy Live uses MySQL for database purpose. So you need to setup mysql::
+
+    $ sudo apt-get install mysql-server
+    $ sudo apt-get install libmysqlclient-dev
+    $ sudo apt-get install python3-dev
+    $ sudo systemctl start mysql
+    $ sudo mysql
+
+This will open mysql. You will now have to create a new database with name 
+{database_name}. Create a new user {user_name} and password {user_password}.
+Grant all previleges to {user_name} on {database_name}::
+
+    mysql> CREATE DATABASE {database_name};
+    mysql> UPDATE mysql.user SET authentication_string = PASSWORD('{user_password}') WHERE User = '{user_name}';
```

Comment:
This doesn't works for me, 
```
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ...
```
I think this syntax varies with mysql version.

Reply:
I have updated the README. If it still doesn't work out for you please check MySQL documentation.